R/tabBooks.R

```diff
@@ -36,14 +36,17 @@ tabBooks <- function(dataset, vars, banner, weight = NULL) {
     # for every "column" variable
     for (vbi in seq_along(book[[vi]])) {
       crunch_cube <- book[[vi]][[vbi]]
+      # there is a very rare case where a user will want to "include missing", should handle that
+      # somewhere in ... or other options to tabBooks.
```

Comment:
it seems that "no" is a default value of the 'useNA' parameter of the 'crtabs' function, so setting it here shouldn't matter